vueui/src/components/view/admin/permissions-panel.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div>
+    <div v-if="forumuserui_enabled" class="notice">
+      <center>Modifications done do not outlast the round!</center>
```

Comment:
You could leave some information on where to perform persistent modifications here.